examples/java/gradle-shared-module/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
-#Wed Dec 18 13:39:13 EST 2019
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
+#Wed Jan 04 09:19:33 PKT 2023
```

Comment:
This is not related, should this even be tracked 🤔

Reply:
Yeah, I think it shouldn't be. I added it because it was already there.